frontend/src/config/index.ts

```diff
@@ -82,7 +82,15 @@ type Country = keyof typeof configMap;
 
 const DEFAULT: Country = 'mozambique';
 
-const { REACT_APP_COUNTRY: COUNTRY } = process.env;
+const {
+  REACT_APP_COUNTRY: COUNTRY,
+  REACT_APP_OAUTH_CLIENT_ID: CLIENT_ID,
+  REACT_APP_OAUTH_AUTHORITY: AUTHORITY,
+  REACT_APP_OAUTH_REDIRECT_URI: REDIRECT_URI,
+  REACT_APP_TESTING: TESTING,
+  REACT_APP_ACLED_API_ISO: ACLED_ISO, // ACLED iso code.
```

Comment:
Could this be a config variable instead of an environment variable? This does not need to be hidden or secret, does it?

Reply:
This could even be in the `acled_incidents` layer definition, no?

Reply:
Then add it to prism.json, no?

Reply:
Agreed, this is a good suggestion. @JorgeMartinezG

Reply:
Done